src/docfx/lib/github/GitHubUser.cs

```diff
@@ -2,12 +2,13 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System;
+using System.Linq;
 
 namespace Microsoft.Docs.Build
 {
     internal class GitHubUser
     {
-        public int Id { get; set; }
+        public int? Id { get; set; }
```

Comment:
can we make `int?` -> `int`, `int?` feels like an overkill

Reply:
want to make the cahce file clean without `"id": 0"

---
In reply to: [226194902](https://github.com/dotnet/docfx/pull/3467#discussion_r226194902) [](ancestors = 226194902)